pybossa/util.py

```diff
@@ -1259,11 +1259,40 @@ def process_table_component(tp_code, user_response):
         if type(response_value) is dict:
             response_value = list(response_value.values())
 
+        # handle existing data in the response
```

Comment:
Looks like there are two parts of data that are to be displayed here under task presenter; task `request fields` and `response fields`. And `request fields` are termed as  `existing data` or `initial_data`. 
Would it make sense to term initial/existing as `request_fields` instead? This will also be inline with task answers being considered as `response_values` here. If so, the comment, and variable names would need update `initial_data` -> `request_values` or something similar that defines them as `request_` ... What do you think?

Reply:
Good suggestion. That actually opens my mind with another thought. So, there are actually three parts need to be displayed in the task presenter:
1. request field (like task.info.contact expressed as ```:initial_data="task.info.contact"```
2. initial data in the table element (like ```:initial_data=["a": 123, "b": 456]```
3. response data

How about I rename them to reflect the above description and use a new term from you comment ```existing_data``` to reflect either "request field" or "initial data"? I will submit a PR to see how it looks and we can further discuss about it :)

Reply:
The changes looks great. 
One minor comment. If there's no reason to use type instead of isinstance, the code can be simplified further with 
replacing
```python
if type(request_fields) is list:
``` 
with
```python
if isinstance(request_fields, list):
```

Similarly, for not operation. Thanks!